.github/workflows/benchmark.yml

```diff
@@ -55,7 +55,7 @@ jobs:
         run: ./run git-clean
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      - run: ./run backend benchmark runtime enso
+      - run: ./run backend benchmark runtime
```

Comment:
Running the stdlib benchmark is important. Especially when they start collecting results using the same way as [engine benchmark runner](https://github.com/enso-org/enso/blob/a7bc3c6c89d07eda08130edfa5dd966b4ab5244d/engine/runtime/src/bench/java/org/enso/interpreter/bench/BenchmarksRunner.java#L58) does. Then their results will be "processable" by the [same performance tools](https://github.com/enso-org/enso/blob/3ca0a17d663ad1c28503438296b6f62f42e08574/tools/performance/engine-benchmarks/README.md).

Then it will be desirable to give the stdlib benchmarks a GitHub Action to run daily.

Reply:
Added https://github.com/enso-org/enso/issues/6554